taichi/backends/codegen_llvm.h

```diff
@@ -495,6 +495,44 @@ class CodeGenLLVM : public IRVisitor, public ModuleBuilder {
       } else {
         TC_NOT_IMPLEMENTED
       }
+    } else if (op == BinaryOpType::pow) {
+      if (current_arch() == Arch::x86_64) {
+        if (ret_type == DataType::f32) {
+          stmt->value =
+              create_call("pow_f32", {stmt->lhs->value, stmt->rhs->value});
+        } else if (ret_type == DataType::f64) {
+          stmt->value =
+              create_call("pow_f64", {stmt->lhs->value, stmt->rhs->value});
+        } else if (ret_type == DataType::i32) {
+          stmt->value =
+              create_call("pow_i32", {stmt->lhs->value, stmt->rhs->value});
+        } else if (ret_type == DataType::i64) {
+          stmt->value =
+              create_call("pow_i64", {stmt->lhs->value, stmt->rhs->value});
+        } else {
+          TC_P(data_type_name(ret_type));
+          TC_NOT_IMPLEMENTED
+        }
+      } else if (current_arch() == Arch::cuda) {
+        if (ret_type == DataType::f32) {
+          stmt->value =
+              create_call("__nv_powf", {stmt->lhs->value, stmt->rhs->value});
+        } else if (ret_type == DataType::f64) {
+          stmt->value =
+              create_call("__nv_pow", {stmt->lhs->value, stmt->rhs->value});
+        } else if (ret_type == DataType::i32) {
+          stmt->value =
+              create_call("nv_pow_i32", {stmt->lhs->value, stmt->rhs->value});
```

Comment:
Looks good! but should here be `pow_i32` instead of `nv_pow_i32`?

Reply:
Yeah, that's it.